airflow/cli/cli_parser.py

```diff
@@ -245,6 +245,9 @@ def string_lower_type(val):
     choices={ColorMode.ON, ColorMode.OFF, ColorMode.AUTO},
     default=ColorMode.AUTO,
 )
+ARG_DAG_ID_REQ_FLAG = Arg(
```

Comment:
Can we not keep just `ARG_DAG_ID` ? Do we need the suffix `REQ_FLAG` ?

Reply:
We do need a suffix, because `ARG_DAG_ID` already exists for the positional dag id arg. I thought this new name was better than `_OPT` since it isn't actually optional.

https://github.com/apache/airflow/blob/73257628219e09c6cfd14b38d0aeed8b6e46a83c/airflow/cli/cli_parser.py#L177